atom/browser/lib/bluetooth_chooser.cc

```diff
@@ -95,15 +95,15 @@ void BluetoothChooser::AddOrUpdateDevice(const std::string& device_id,
                                          bool is_gatt_connected,
                                          bool is_paired,
                                          int signal_strength_level) {
-  DeviceInfo info = {device_id, device_name};
+  DeviceInfo info{device_id, device_name};
```

Comment:
According to our coding style, using uniform initialization is not encouraged in this case:
https://www.chromium.org/developers/coding-style/cpp-dos-and-donts?pli=1#TOC-Variable-initialization